code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -210,6 +210,20 @@ There are several things that need to be remembered:
 				feature_y_offset = glove_offset["y"]
 
 		gloves_overlay.pixel_y += feature_y_offset
+
+		// We dont have any >2 hands human species (and likely wont ever), so theres no point in splitting this because:
+		// It will only run if the left hand OR the right hand is missing, and it wont run if both are missing because you cant wear gloves with no arms
+		// (unless admins mess with this then its their fault)
+		if(num_hands < default_num_hands)
+			var/left_hand_missing = !has_left_hand(check_disabled = FALSE)
```

Comment:
instead of storing this as a bool just store the letter extension to use, no? or is the caps bit important here